src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -141,7 +141,7 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
 };
 
 const SuspectedLocation: React.FC<{ user: WithId<User> }> = ({ user }) => {
-  useFirestoreConnect("venues");
+  useSparkleFirestoreConnect("venues");
```

Comment:
[OOS] Not relevant to this PR, but calling this out as a pattern we want to remove, as we shouldn't ever be loading all of the venues like this.